docs/versioned_docs/version-1.14/reference/builtin-functions/feel-built-in-functions-context.md

```diff
@@ -22,11 +22,11 @@ get value({foo: 123}, "foo")
 Returns the entries of the context as list of key-value-pairs.
 
 * parameters:
-  * `context`: context
+  * `m`: context
 * result: list of context which contains two entries for "key" and "value"
 
 ```js
-get entries({foo: 123})
+get entries(m:{foo: 123})
```

Comment:
:+1: Thank you for updating the docs!

Reply:
u're welcome: I thought it was better to update them for the sake of completeness 😄